### ISSUE TITLE
Removing aria-expanded from menu buttons should not be the recommended approach

### ISSUE THREAD

**Issue by katiehockman** [w3c/aria-practices#586]
"When the menu is displayed, the element with role button has aria-expanded set to true. When the menu is hidden, it is recommended that aria-expanded is not present. If aria-expanded is specified when the menu is hidden, it is set to false." https://www.w3.org/TR/wai-aria-practices/#menubutton

Having a default value of aria-expanded="false" for collapsed menus would more clearly indicate to users that the menu is not currently expanded. This should be preferred/recommended over simply removing aria-expanded.

**Comment by mcking65** [comment]
@katiehockman, this was thoroughly discussed in issue #326. Please review all the comments in that issue. 

In addition to the comments about screen reader verbosity, you might find my comment in that issue about the difference between the menubutton pattern and the disclosure pattern to be helpful.

If issue #326 does not address your questions, feel free to re-open this one with your specific concerns.

**Comment by katiehockman** [comment]
Great, thanks for the link. Read it through, and the reasoning behind not recommending aria-expanded makes sense.
